src/test/testdata/lsl/charLiterals.lsl

```diff
@@ -0,0 +1,13 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias Char = char;
+
+automaton A : Int {
+    var ch1: Char = '\u0041';
+    var ch2: Char = 'a';
+    var ch3: Char = '\u72D0';
+    var ch4: Char = '\52';
+    var ch5: Char = '\123';
+    var ch6: Char = '狐';
+}
```

Comment:
I suggest also adding a test case for non-printable characters. Like `\n`, `\r`, `\0` and such.

Reply:
Fixed;